.github/workflows/release.yml

```diff
@@ -107,103 +107,105 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - name: Setup dependencies
-        run: docker-compose up -d
+        run: docker-compose up -d veritable-agent ipfs opa
       - name: Install Node Packages
         run: npm ci
       - name: Run tests
```

Comment:
uses healthcheck for loading in the correct sequence

Reply:
apparently the github runner has an outdate version of compose that doesn't support `--wait`

Currently, this is not an issue because of the healthchecks we have. If it becomes one, we can add a delay in.